pkg/scalers/azure_pipelines_scaler.go

```diff
@@ -139,7 +139,7 @@ func (s *azurePipelinesScaler) GetAzurePipelinesQueueLength(ctx context.Context)
 	r.Body.Close()
 
 	if !(r.StatusCode >= 200 && r.StatusCode <= 299) {
-		return -1, fmt.Errorf("azure Devops REST api returned error. status: %d response: %s", r.StatusCode, string(b))
+		return -1, fmt.Errorf("api REST of Azure DevOps returned error. url: %s status: %d response: %s", url, r.StatusCode, string(b))
```

Comment:
Is there a reason why the ordering was changed?
```suggestion
		return -1, fmt.Errorf("Azure DevOps REST API returned error. url: %s status: %d response: %s", url, r.StatusCode, string(b))
```

Reply:
Any  thoughts @zroubalik ?

Reply:
I was reading (https://github.com/kedacore/keda/blob/main/pkg/scalers/aws_sqs_queue_scaler.go) they use another class to make errors logs, and now can bypass the ST1005. Is good think?

Reply:
Do you mean `sqsQueueLog`? That is the own scaler logger, but apart from it the errors which block the execution, should be returned as function result. In this case :
```golang
meta.targetQueueLength = defaultTargetQueueLength

	if val, ok := config.TriggerMetadata["queueLength"]; ok && val != "" {
		queueLength, err := strconv.Atoi(val)
		if err != nil {
			meta.targetQueueLength = targetQueueLengthDefault
			sqsQueueLog.Error(err, "Error parsing SQS queue metadata queueLength, using default %n", targetQueueLengthDefault)
		} else {
			meta.targetQueueLength = queueLength
		}
	}
```
The variable `targetQueueLength` already has a default value. Probably, we should review AWS Scalers to make them consistency with the "error handling way", but at least that case will not produce a panic :)

One way to avoid the linting problem could be doing something like, `the Azure Devops REST api returned error`
WDYT?

Reply:
It's ok for me, do you agree @tomkerkhove, if yes, i will make the modification. :)

Reply:
Fine by me, as long as you use DevOps :p